zk-sdk/src/encryption/auth_encryption.rs

```diff
@@ -148,6 +156,7 @@ impl AeKey {
     }
 }
 
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
So just so I understand, these are *not* included because it makes the wasm binary too big, correct? if so, can you add a comment to that effect? otherwise we might accidentally add it in the future

Reply:
That's right. It is to prevent bloat in the binary. I added a comment where we import the dependencies. If we do accidentally remove this attribute in the future, it will not compile since `solana-sdk` is set to `#[cfg(not(target_arch = "wasm32"))]`.